tests/integration/__init__.py

```diff
@@ -448,6 +448,11 @@ def setUp(self):
         self.useFixture(self.test_store)
         self.useFixture(fixtures.EnvironmentVariable("SNAPCRAFT_TEST_INPUT", "1"))
 
+        # Ensure TTY always appears connected.
```

Comment:
I did not spot this one before, this will not work, you cannot patch in integration tests as they run from the built snap.